request_public_data.py

```diff
@@ -23,16 +25,19 @@
 if args.verbose:
 	logging.basicConfig(level=logging.DEBUG)
 
-r = requests.get("http://" + server_url + "/getData?from_date=" + from_date + "&to_date=" + to_date)
+from_date_UTC = arrow.get(from_date).replace(tzinfo='local').to('utc').format('YYYY-MM-DD HH:mm:ss')
```

Comment:
yea, I didn't use strptime because I thought one of the advantage of using arrow is that it parses a string to a date/arrow object directly. However, if you use arrow.get() to parse a string then you cannot specify the `tzinfo` at creation time (it converts to UTC by default), but you can use the replace function to set `tzinfo`.
I have tested this out in ipython notebook, and this does work: 
#### input:
```python
from_date = '2016-07-12'
to_date = '2016-07-13'

from_date_UTC = arrow.get(from_date).replace(tzinfo='local').to('utc').format('YYYY-MM-DD HH:mm:ss')
to_date_UTC = arrow.get(to_date).replace(tzinfo='local').to('utc').format('YYYY-MM-DD HH:mm:ss')
print from_date_UTC
print to_date_UTC

from_ts = arrow.get(from_date_UTC).timestamp
to_ts = arrow.get(to_date_UTC).timestamp
print from_ts
print to_ts

from_dt = arrow.get(from_ts)
to_dt = arrow.get(to_ts)
print from_dt
print to_dt
```   

#### output:  
```
2016-07-12 07:00:00
2016-07-13 07:00:00
1468306800
1468393200
2016-07-12T07:00:00+00:00
2016-07-13T07:00:00+00:00
```